templates/aws-stack.yml

```diff
@@ -593,6 +578,14 @@ Resources:
       SubnetId: !Ref Subnet1
       RouteTableId: !Ref Routes
 
+  BuildkiteAgentTokenParameter:
+    Type: AWS::SSM::Parameter
+    Condition: CreateAgentTokenParameter
+    Properties:
+      Name: !Sub "/${AWS::StackName}/buildkite/agent-token"
```

Comment:
I might rename this `/${AS::StackName}/buildkite-agent/token`, thoughts?

Reply:
Not really sure what is the benefit to make this stack dependent, as most of the use cases, we will have one token for several bk stacks. BTW, if we have two organizations within the same AWS account, the user can easily customize this value by not using the default one.

We have used `/vendors/buildkite/agent-token` and it works for us really well.

Reply:
This resource is only created if you don’t specify your own SSM parameter path. If this stack were to create a global resource _not stack scoped_ it would conflict with subsequent deployments of the same CloudFormation template.